test/conformance/chainsaw/validate/clusterpolicy/cornercases/two-rules-with-different-action/chainsaw-test.yaml

```diff
@@ -6,23 +6,29 @@ metadata:
 spec:
   steps:
     - name: step-01
+      try:
+      - script:
+          content: kubectl patch configmap kyverno -p '{"data":{"generateSuccessEvents":"true"}}' -n kyverno
```

Comment:
this looks like it its overwriting data variable in the configmap, I think you should use jsonpatch add operation in patch